package/packages.yaml

```diff
@@ -26,169 +26,209 @@ local:
         name: ContingentClaims.Core
         path: main/daml/ContingentClaims.Core
         base-module: ContingentClaims.Core
+        increment-version: patch
```

Comment:
I think the increment depends on the nature of the change at hand. I guess on a subsequent version this could be an interactive input to the command line

Reply:
Yeah, that's a good idea